src/orange/orangeinp/ConvexRegion.hh

```diff
@@ -230,6 +230,60 @@ class InfWedge final : public ConvexRegionInterface
     Turn interior_;
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * A general parallelepiped centered on the origin.
+ *
+ * A parallelepiped is a shape having 3 pairs of parallel faces out of
+ * which one is parallel with the XY plane (Z faces). All faces are
+ * parallelograms in the general case. The Z faces have 2 edges parallel
+ * with the X-axis.
+ *
+ * The shape has the center in the origin and it is defined by:
+ *
+ *   - `dX, dY, dZ:` half-lengths of the projections of the edges on X, Y
+ *     and Z. The lower Z face is positioned at `-dZ`, while the upper at
+ * `+dZ`.
+ *   - `alpha:` angle between the segment defined by the centers of the
+ *     X-parallel edges and Y axis `[-90,90]` in degrees
+ *   -   `theta:` theta angle of the segment defined by the centers of the Z
+ * faces;
+ *   - `phi:` phi angle of the same segment
```

Comment:
It looks like clang-format messed up the indentation here; could you fix?

Reply:
Fixed.